docs/manual/src/docs/asciidoc/_includes/servlet/authentication/rememberme.adoc

```diff
@@ -51,7 +51,7 @@ If you have more than one in your application context, you need to specify which
 
 [[remember-me-persistent-token]]
 === Persistent Token Approach
-This approach is based on the article http://jaspan.com/improved_persistent_login_cookie_best_practice[http://jaspan.com/improved_persistent_login_cookie_best_practice] with some minor modifications  footnote:[Essentially, the username is not included in the cookie, to prevent exposing a valid login name unecessarily.
+This approach is based on the article https://web.archive.org/web/20180819014446/http://jaspan.com/improved_persistent_login_cookie_best_practice[http://jaspan.com/improved_persistent_login_cookie_best_practice] with some minor modifications  footnote:[Essentially, the username is not included in the cookie, to prevent exposing a valid login name unecessarily.
```

Comment:
This link should be updated in the javadoc also in the `PersistentTokenBasedRememberMeServices` class

Reply:
It might be worth while to document the algorithm implemented by spring security in the docs, rather than pointing to web.archive.org as that link might disappear at some point in the future.

Reply:
@asaikali This is a good suggestion, but I think in this PR we should stick to fixing the broken links. Could you please open an issue specifically to document it in the docs instead of point out to the web.archive.org domain?